board/piksiv3/rootfs-overlay/etc/init.d/S86skylark1_upload_daemon

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
Can scripts at the same init level start in parallel? Seems like this might create a race condition between the setup script and the daemons